community/modules/compute/schedmd-slurm-gcp-v5-partition/README.md

```diff
@@ -32,6 +32,51 @@ The following code snippet creates a partition module with:
 For a complete example using this module, see
 [slurm-gcp-v5-cluster.yaml](../../../examples/slurm-gcp-v5-cluster.yaml).
 
+### Compute VM Zone Policies
+
+> **_WARNING:_** Lenient zone policies can lead to additional egress costs when
+> moving data between Google Cloud resources in different zones in the same
+> region, such as between filestore and other VM instances. For more information
+> on egress fees, see the [Network Pricing][networkpricing] Google Cloud
+> documentation.
+>
+> To avoid egress charges, ensure your compute nodes are created in the same
+> zone as the other resources that share data with them by setting
+> `zone_policy_deny` to all other zones in the region.
+
+The Slurm on GCP partition modules provide the option to set policies regarding
+which zone the compute VM instances will be created in through the
+`zone_policy_allow` and `zone_policy_deny` variables.
+
+As an example, see the the following module: will show preference for zones
```

Comment:
I would not use "preference" here but a more neutral word like allow.

Reply:
Unless this actually causes those zone to be more highly ranked than 1-c.

Reply:
It was a mistake to include that part, everything after the : should have been removed in favor of the bulleted list below. It's been fixed.